lessons/Early-Math/commutative-property/langs/en-us/pages/comm-prop-table.adoc

```diff
@@ -0,0 +1,55 @@
+[.landscape]
+= Commutative Property Table
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+
+For each Circle, apply the Commutative Property as many times as you can.
+
+[.FillVerticalSpace,cols=".^1a,^.^6a,^.^6a,^.^6a,^.^6a", stripes="none", options="header"]
+|===
+|	 | Analyze this Circle. | Apply the Commutative Property. | Apply it again. | Apply it again?
```

Comment:
What do you actually want them to do in the part that says Analyze this Circle. 
I would probably label this column "Begin with this Circle of Evaluation" or "Original Expression"